### DISCUSSION TITLE
Custom tags help

### DISCUSSION THREAD

**Discussion by G0nz0uk** [telegraf/telegraf#1396] [Q&A] [answered]
Hello,

I've been working on this today without success:
https://www.reddit.com/r/grafana/comments/m0gfkb/global_tags_in_telegrafconf_screenshots_to_help/

I have Telegraf, Influxdb and Grafana on the same VM that is using individual telegraf.conf files in the telegraf.d folder to pull snmp info from different sites.  I need to tag these site locations so I can sort in Grafana.

I tried the [global_tags] command in each .conf file and it seems one overwrites the other (screenshot in reddit link).

I use the [[inputs.snmp]] in each .conf is there a way to tag in there?

I did try:
>     [[inputs.snmp]]
>       agents = [ "10.20.30.1:161" ]
>       version = 2
>       community = "public"
>       [inputs.snmp.tags]
>          location="New York"

But telegraf would no longer start.

Any help would be most appreciated.

Thanks

**Comment by abhi-jha** [comment, marked as answer]
I don't think this is the right project for this query. 

It should go to : https://github.com/influxdata/telegraf/
